Compiler/Template/CodegenCFunctions.tpl

```diff
@@ -4098,9 +4098,9 @@ template crefToCStr(ComponentRef cr, Integer ix, Boolean isPre, Boolean isStart)
   case SIMVAR(varKind=DAE_RESIDUAL_VAR())
   case SIMVAR(index=-2)
   then
-    if intEq(ix,0) then (if isPre then "$P$PRE")+crefDefine(cr)
-    else '_<%if isPre then "$P$PRE"%><%crefDefine(cr)%>(<%ix%>)'
-  case var as SIMVAR(index=-1) then error(sourceInfo(), 'crefToCStr got index=-1 for <%variabilityString(varKind)%> <%crefStr(name)%>')
+    if intEq(ix,0) then (if isPre then "$P$PRE")+crefDefine(cr)+' /* <%escapeCComments(crefStrNoUnderscore(cr))%>*/'
+    else '_<%if isPre then "$P$PRE"%><%crefDefine(cr)%>(<%ix%>) /* <%escapeCComments(crefStrNoUnderscore(cr))%>*/'
```

Comment:
these two lines generate nested comments...

Reply:
Yeah, we seem to use that code is quite weird ways although we shouldn't use #define for variables if possible (*looks over at @wibraun*)... I'll just remove those parts :(

Reply:
Yeah, agree. Actually I already improved last week the variable creation for jacobian, so now we just need adapt the code generation.